ci-operator/step-registry/janus-idp/backstage-showcase/helm-aks-nightly-mapt-create/OWNERS

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/janus-idp/backstage-showcase root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- josephca
+- rnapoles-rh
```

Comment:
I suggest removing 'rnapoles-rh' from all OWNERS files as he is no longer working for the project. Thanks!

Reply:
That makes sense. I can incorporate it in this PR.